survey_crm_generation/tests/test_survey_crm_sale_generation.py

```diff
@@ -37,8 +37,10 @@ def test_lead_generation(self):
             ),
         )
         expected_lead_description = (
-            "E-mail address: test@test.com\n"
+            "<p>E-mail address: test@test.com\n"
             "Your company name?: Tecnativa\n"
-            "And your name?: Tecnativa"
+            "And your name?: Tecnativa</p>"
+        )
+        self.assertEqual(
+            str(self.generated_lead.description), expected_lead_description
```

Comment:
Better convert to a Markup object the expected_lead description

Reply:
Done, thanks for the suggestion